tests/API1/testfileselector.py

```diff
@@ -0,0 +1,176 @@
+# import os
+# import shutil
+# import tempfile
+
+# import param
+
+# from . import API1TestCase
+
+
+# class TestFileSelectorParameters(API1TestCase):
```

Comment:
Why is this commented out?

Reply:
Arf good catch this wasn't mean to be committed, I'm working on fixing an issue with FileSelector and adding some tests, I'll remove that file now.